addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -348,6 +348,13 @@ export class GraphRenderer extends Component {
             if (stacked) {
                 dataset.backgroundColor = hexToRGBA(dataset.borderColor, 0.4);
             }
+            if (cumulated) {
+                let accumulator = 0;
+                dataset.data = dataset.data.map((value) => {
+                    accumulator += value;
+                    return accumulator;
```

Comment:
You should use .reduce :)

Reply:
I though so but no, it's a map and reduce